modules/ingest-geoip/build.gradle

```diff
@@ -26,7 +26,7 @@ dependencies {
   // geoip2 dependencies:
   implementation("com.fasterxml.jackson.core:jackson-annotations:2.13.1")
   implementation("com.fasterxml.jackson.core:jackson-databind:2.13.1")
-  runtimeOnly("com.fasterxml.jackson.core:jackson-core:2.13.1")
+  implementation("com.fasterxml.jackson.core:jackson-core:2.13.1")
```

Comment:
This is unfortunate since it means jackson will now show up in eg autocomplete again when working within ingest-geoip classes :/

Reply:
Yeah, this is not correct. Let me revisit this.